artifacts/contracts/oz/access-control/AccessControlContract.sol/AccessControlContract.json

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
Hmm have you rebased the main branch? Before the 0.7 release, I made sure that the "oz" folder has been moved to "openzeppelin" folder. These metadata files shouldn't be here

Reply:
Hmm these metadata files are still here. I mean if there's only 1 new/modified contract should result into 1 new metadata file.